pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -112,6 +113,7 @@ func (fr *fileReader) startWalkingDisk() error {
 			return err
 		}
 	}
+	fr.Path = path
```

Comment:
Wouldn't this cause problem when the symlinked folder change? Since `fr.Path` now is the resolved path.

Reply:
fr.Path is updated every time when startWalkingDisk gets called. I have two paths variable because one is the abs path if created to always try to see if theres a symlink and resolves and sets it in fr.Path which is used in go tests to pass the test. I know you mentioned passing path in startWalkingDisk , I can either leave the function how it was before. just wasnt sure how to give back resolved path to pass go tests. so hence why i made this change.

Reply:
I don't think we should set that value just to satisfy the tests. Kind tricky to understand the purpose when looking at the code. Maybe we can extract the path validation to another function and just test that instead. Or if you have another suggestion for how we can test this behavior.